Doc/library/dataclasses.rst

```diff
@@ -188,6 +188,13 @@ Module contents
 
     .. versionadded:: 3.10
 
+    .. versionchanged:: 3.11
+       If a field name is already included in the ``__slots__``
+       of a base class, it will not be included in the generated ``__slots__``.
```

Comment:
Can you add a pointer to the `__slots__` documentation that you mentioned in the original bpo post?

Reply:
I've added a link to the docs, is that what you mean?

Reply:
Yes.